# learn_telebot
### Изучение библеотеки telebot

Во аремя изучения телебот я столкнулся с одной неприятной ошибкой. Когда я пытаюсь передать в register_next_step_handler дополнительные аргументы - всё вылетает сс NameError. Помогите пожалуйста)

![image](https://user-images.githubusercontent.com/91367326/180640212-aafc061f-32b1-46b2-9397-24b5574bd4c7.png)
![scrin](https://user-images.githubusercontent.com/91367326/180640584-c478be2c-c0c7-4af3-a88b-ec94c144f64a.png)
